dts/arm/st/f1/stm32f100Xe.dtsi

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 Jonas Eriksson, Up to Code AB
+ *
+ * SoC device tree include for STM32F100xE SoCs
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/f1/stm32f100b.dtsi>
```

Comment:
stm32f100**X**b

Reply:
Damnit, I must have forgotten to clear my build cache when testing. Fixed now